pom.xml

```diff
@@ -212,7 +212,7 @@
             <dependency>
                 <groupId>com.fasterxml.jackson.core</groupId>
                 <artifactId>jackson-databind</artifactId>
-                <version>${jackson-version}</version>
+                <version>${jackson-version}.2</version>
```

Comment:
I was going to suggest making the entire version explicit, instead of using the variable with ".2" appended.  But no matter how you do this, it's fragile. I'd suggest putting an XML comment right before the `<jackson-version>` definition on line 454, noting that this is overridden for `jackson-databind`, and pointing to this PR for background.

Reply:
@tedepstein I don't see this as fragile at all. This is the only time I've ever seen Jackson publish anything with a four-part version, and I'm sure it's all about this security fix. If we bump to a different Jackson version, the chance that (a) there will be a corresponding `.2` version for jackson-databind, and that it will not be fully compatible with the non-`.2` versions of the other libraries, seems minuscule. Far more likely is that the build will fail, and this will be the obvious culprit. At most, there'd be a tiny bit of head-scratching by whoever runs into it, before they just get rid of the `.2`.

Reply:
@tedepstein BTW, making the version explicit WOULD be fragile, since it wouldn't lead to the expected failure if the property were changed.

Reply:
You're right @andylowry , it's better to fail loudly. I'll resolve this conversation and you can feel free to merge, or wait for @ghillairet's review.